templates/kubernetes/features.html

```diff
@@ -161,8 +161,10 @@ <h2>Logging and Monitoring</h2>
 </section>
 <section class="p-strip--light">
   <div class="row">
+    <h2>Get started with Canonical&rsquo;s Distribution of Kubernetes<sup>&reg;</sup></h2>
     <p>Find out how the Canonical Distribution of Kubernetes can give you the flexibility to share operational code with a large community whilst placing your services exactly where you want them.</p>
```

Comment:
Copy should be "Find out how Canonical can enable Kubernetes on demand for your devops teams -- on OpenStack, on VMware, on public clouds, and on bare metal clusters."